packages/contracts/test/dollar/core/StakingShare.t.sol

```diff
@@ -153,33 +175,33 @@ contract RemoteDepositStateTest is DepositState {
         emit Paused(admin);
 
         vm.prank(admin);
-        staking.pause();
+        IAccessCtrl.pause();
     }
 
     function testPause_ShouldRevert_IfNotPauser() public {
-        vm.expectRevert("not pauser");
+        vm.expectRevert("MGR: Caller is not admin");
         vm.prank(secondAccount);
-        staking.pause();
+        IAccessCtrl.pause();
     }
 
     function testUnpause_ShouldUnpause() public {
         vm.prank(admin);
-        staking.pause();
+        IAccessCtrl.pause();
 
         vm.expectEmit(true, false, false, true);
         emit Unpaused(admin);
 
         vm.prank(admin);
-        staking.unpause();
+        IAccessCtrl.unpause();
     }
 
     function testUnpause_ShouldRevert_IfNotPauser() public {
         vm.prank(admin);
-        staking.pause();
+        IAccessCtrl.pause();
 
-        vm.expectRevert("not pauser");
+        vm.expectRevert("MGR: Caller is not admin");
```

Comment:
```suggestion
        vm.expectRevert("Manager: Caller is not admin");
```

We should not shorten words unnecessarily, unless there is a reason for shortened revert messages.

Reply:
gas saving is the reason :smile:

Reply:
If its only gas saving for deployment I think its not worth it. If it saves gas for transactions then it is worth it.

Reply:
ok will change